lms/static/sass/discussion/_discussion.scss

```diff
@@ -52,8 +40,8 @@ body.discussion {
     }
 
     .edit-post-title {
-      @include box-sizing(border-box);
-      border: 1px solid #aaa;
+      box-sizing: border-box;
```

Comment:
I think the `@include` uses Bourbon and automatically spits out different rules for various browsers.